src/pl/tcl/meson.build

```diff
@@ -0,0 +1,55 @@
+if not tcl_dep.found()
+  subdir_done()
+endif
+
+pltcl_sources = files(
+  'pltcl.c',
+)
+
+gen_pltclerrcodes = files('generate-pltclerrcodes.pl')
+pltcl_sources += custom_target('pltclerrcodes.h',
+  input: files('../../backend/utils/errcodes.txt'),
+  output: 'pltclerrcodes.h',
+  capture: true,
+  command: [perl, gen_pltclerrcodes, '@INPUT@']
+)
+
+shared_module('pltcl',
+  pltcl_sources,
+  c_pch: '../../include/pch/postgres_pch.h',
+  include_directories: [include_directories('.'), postgres_inc],
+  kwargs: pg_mod_args + {
+    'dependencies': [tcl_dep, pg_mod_args['dependencies']],
+  },
+)
+
+install_data(
+  'pltcl.control',
+  'pltcl--1.0.sql',
+  install_dir: get_option('datadir') / 'extension'
+)
+
+install_data(
+  'pltclu.control',
+  'pltclu--1.0.sql',
+  install_dir: get_option('datadir') / 'extension'
+)
```

Comment:
I'd probably just make those one install_data() call. But it doesn't matter much.